2.8.1 Breaking Changes - Module not found: Default condition should be last one

### [REQUIRED] Environment
- Alchemy SDK version: 2.8.1
- typescript: 5.0.4
- next: 13.3.1
### [REQUIRED] Describe the problem
After updating from 2.8.0 to 2.8.1 app will no longer build. 
tsconfig.json has "moduleResolution": "node" as nextjs forces module resolution to node. 
I saw in the release notes for 2.8.1 there was a fix for the SDK previously not being compatible with bundler. It appears that has potentially broken previously compatible moduleResolutions. 

```
Module not found: Default condition should be last one
> 1 | import { Alchemy, Network } from "alchemy-sdk";
```

#### How to reproduce:
Upgrade from 2.8.0 to 2.8.1 in a nextjs environment 
Import the alchemy sdk


#### Relevant code or sample repro:

Module not found: Default condition should be last one

I solved this by manually going into node_modules/alchemy-sdk/package.json. :

Line 26: 35::
 "exports": {
    "node": {
      "types": "./dist/src/index.d.ts",
      "require": "./dist/cjs/index.js",
      "import": "./dist/cjs/index.js"
    },
    "package.json": "./package.json",
    "types": "./dist/src/index.d.ts",
    "default": "./dist/esm/index.js"
  },

Make sure "default": 
is the last line of the export.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,8 @@
 
 ### Minor Changes
 
+- Moved the `exports.default` field to be the last field. This fixed a bug introduced in `2.8.1` that prevented certain frameworks from building (#313).
+
 ## 2.8.1
 
 ### Major Changes

--- a/package.json
+++ b/package.json
@@ -30,8 +30,8 @@
       "import": "./dist/cjs/index.js"
     },
     "package.json": "./package.json",
-    "default": "./dist/esm/index.js",
-    "types": "./dist/src/index.d.ts"
+    "types": "./dist/src/index.d.ts",
+    "default": "./dist/esm/index.js"
   },
   "types": "./dist/src/index.d.ts",
   "repository": {

